vault_core/src/vault.rs

```diff
@@ -55,8 +55,11 @@ pub struct Vault {
     /// The total number of tokens deposited
     tokens_deposited: PodU64,
 
-    /// Max capacity of tokens in the vault
-    capacity: PodU64,
+    /// Maximum capacity of supported mint tokens that can be deposited
+    /// after deposit_capacity is reached, no more deposits/minting of
+    /// VRTs can occur. However, the vault can hold more supported tokens
+    /// than the deposit capacity, but no more VRTs can be minted.
```

Comment:
The maximum deposit capacity allowed during VRT minting operations.
The deposited assets in the vault may be exceed the deposit_capacity during non-minting operations, such as vault balance updates.

Reply:
Done